frontend/src/container/PanelWrapper/PiePanelWrapper.tsx

```diff
@@ -0,0 +1,220 @@
+			{pieChartData.length > 0 && (
+				<>
+					<div className="piechart-container" ref={chartRef}>
+						<svg width={width} height={height} ref={containerRef}>
+							<Group top={height / 2} left={width / 2}>
+								<Pie
+									data={pieChartData}
+									pieValue={(data: {
+										label: string;
+										value: string;
+										color: string;
+									}): number => parseFloat(data.value)}
+									outerRadius={({ data }): number => {
+										if (!active) return half - 3;
+										return data.label === active.label ? half : half - 3;
+									}}
+									padAngle={0.02}
+									cornerRadius={3}
+									width={size}
+									height={size}
+								>
+									{
+										// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+										(pie) =>
+											pie.arcs.map((arc, index) => {
+												const { label } = arc.data;
+												const [centroidX, centroidY] = pie.path.centroid(arc);
+												const hasSpaceForLabel = arc.endAngle - arc.startAngle >= 0.6;
+												const arcPath = pie.path(arc);
+												const arcFill = arc.data.color;
+												return (
+													<g
+														// eslint-disable-next-line react/no-array-index-key
+														key={`arc-${label}-${index}`}
+														onMouseEnter={(): void => {
+															showTooltip({
+																tooltipData: {
+																	label,
+																	value: arc.data.value,
+																	color: arc.data.color,
+																	key: label,
+																},
+																tooltipTop: centroidY + height / 2,
+																tooltipLeft: centroidX + width / 2,
+															});
+															setActive(arc.data);
+														}}
+														onMouseLeave={(): void => {
+															hideTooltip();
+															setActive(null);
+														}}
+													>
+														<path d={arcPath || ''} fill={getFillColor(arcFill)} />
+														{hasSpaceForLabel && (
+															<text
+																x={centroidX}
+																y={centroidY}
+																dy=".33em"
+																fill="#000"
+																fontSize={10}
+																textAnchor="middle"
+																pointerEvents="none"
+															>
+																{arc.data.label}
+															</text>
+														)}
+													</g>
+												);
+											})
+									}
+								</Pie>
+							</Group>
+						</svg>
+						{tooltipOpen && tooltipData && (
+							<TooltipInPortal
+								top={tooltipTop}
+								left={tooltipLeft}
+								style={{
+									...tooltipStyles,
+									background: isDarkMode ? Color.BG_INK_400 : Color.BG_VANILLA_100,
+									color: isDarkMode ? Color.BG_VANILLA_100 : Color.BG_INK_400,
+								}}
+							>
+								<div
+									style={{
+										background: tooltipData.color,
+										width: '15px',
+										height: '3px',
+										borderRadius: '2px',
+									}}
+								/>
+								{tooltipData.key}
+								<div>
```

Comment:
We can use div with a className and can get rid of strong.

Reply:
Done,